packages/database/compat/index.node.ts

```diff
@@ -0,0 +1,153 @@
+/**
```

Comment:
Seems like index.node.ts and index.ts can share some code but don't know if it's worth worrying about that at this time.